src/routes/route.tsx

```
@@ -1,10 +1,12 @@
 import { createBrowserRouter } from 'react-router-dom';
 
 import { App } from '@/App';
+import { Map } from '@/pages/Map';
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)